grafana-plugin/src/components/SourceCode/SourceCode.module.scss

```diff
@@ -1,29 +1,29 @@
 .root {
   position: relative;
   width: 100%;
+
+  &:hover .button {
+    opacity: 1;
+  }
 }
 
 .scroller {
   overflow-y: auto;
-}
 
-.scroller_max-height {
-  max-height: 400px;
+  &--maxHeight {
+    max-height: 400px;
+  }
 }
 
-.root .button {
+.button {
   position: absolute;
   top: 15px;
   right: 15px;
   opacity: 0;
   transition: opacity 0.2s ease;
 }
 
-.root:hover .button {
-  opacity: 1;
-}
-
-.root pre {
+pre {
```

Comment:
don't add global styles to particular module.css please, the only place to add global styles is `grafanaGlobalStyles.css`

Reply:
Hmm, that's a mistake I see. it should've been `.root pre` because it was already existing